src/bap_mc/bap_mc.ml

```diff
@@ -170,6 +201,22 @@ module Cmdline = struct
     Arg.(value & opt_all ~vopt:`bil (enum formats) [] &
          info ["show-bil"] ~doc)
 
+  let addr_offset =
+    let doc = sprintf
```

Comment:
This is not an offset, it is an address of a fist byte. Better name would be just `--addr`